public/js/main.js

```diff
@@ -1,6 +1,6 @@
 var bingo = angular.module('bingo', ['ngRoute', 'btford.socket-io', 'ngMaterial'])
   .factory('bingosockets', function(socketFactory) {
-    var myIoSocket = io.connect('http://localhost:3000');
+    var myIoSocket = io.connect('https://bingojs.herokuapp.com');
```

Comment:
Ah, so it would. I'll look into this when I'm back in linux. We might be able to make it work with just `io.connect()`.